webpack/components/extensions/HostDetails/DetailsTabCards/HwPropertiesCard.js

```diff
@@ -27,11 +31,11 @@ const HostDisks = ({ totalDisks }) => {
 };
 
 HostDisks.propTypes = {
-  totalDisks: PropTypes.number,
+  blockDevices: PropTypes.string,
 };
 
 HostDisks.defaultProps = {
-  totalDisks: null,
+  blockDevices: null,
```

Comment:
If `blockDevices` comes in as `null`, this will cause a PropTypes error. Should set the default to `''` here and then make sure that's handled-- _or_ make it a required prop.